notebooks/metadata_example.yaml

```diff
@@ -0,0 +1,74 @@
+NWBFile:
+  session_description: Description.
+  session_start_time: 2000-07-23T00:00:00-07:00
+  institution: My University
+  lab: My Lab
+  experimenter:
+    - Last, First Middle
+Subject:
+  subject_id: subid012345
+  genotype: not defined
+  description: not defined
+  species: Mus musculus
+  date_of_birth: 2023-03-17T00:00:00
+  sex: F
+Ophys:
+  FiberPhotometry:
+    FiberPhotometryTable:
+      location: Location of this fiber
+      coordinates: [0.0, 0.0, 0.0] # Fiber placement taxonomical coordinates in (AP, ML, Z) in mm relative to Bregma
+      notes: extra notes
+      indicator: indicator
```

Comment:
shouldn't this be a list of entries?
e.g. 

```yaml
Ophys:
  FiberPhotometry:
    FiberPhotometryTable:
      - location: SNc
        coordinates: [-3.25, 1.55, -3.8]
        indicator: GCaMP6f
        optical_fiber: chGreen,
        excitation_source: ExcitationSource470,
        photodetector: PhotodetectorPMT,
        dichroic_mirror: dichroic0,
        emission_filter: optical_filter_1,
    FiberPhotometryResponseSeries:
      - name: FiberPhotometryResponseSeriesGreen  # or FiberPhotometryResponseSeriesGCaMP6?
        description: Green fluorescence collected during 470 nm illumination re-binned to 100 Hz
        fiber_photometry_table_region: [0]
```

To be able to identify which excitation source device etc. to create for each `FiberPhotometryResponseSeries`